src/sv_demo.c

```diff
@@ -1832,10 +1832,10 @@ const char* SV_MVDDemoName(void)
 		if (d->desttype == DEST_STREAM) {
 			continue; // streams are not saved on to HDD, so ignore it...
 		}
-		if (d->name && d->name[0]) {
+		if (d->name[0] != 0) {
```

Comment:
I'd rather have `'\0'` here 😃

Reply:
Have updated this. Yeah I like that better, too. Thanks!

Reply:
But if d->name is NULL, then core dumps.

Reply:
https://github.com/deurk/mvdsv/blob/5fb8cc4255100990e22181ab38f2028b7e3c6f9d/src/server.h#L437 < d->name is an array not a pointer (this is why the compiler was issuing a warning)